modules/tutorials/partials/migration-tutorial/student-database-example.adoc

```diff
@@ -0,0 +1,33 @@
+[#student-record-sql-database-section]
+== Student Record database
+
+The database we will convert will consist of a relational structure with three tables:
+
+[plantuml,student-record-erd]
+.Student records SQL database
+....
+include::partial$diagrams/student-record-erd.puml[]
+....
+
+which we will convert to a document model suitable for storage in our Couchbase bucket:
```

Comment:
Could we change this so that this isolated line starts with a capital letter:

We will convert this table structure to a document model suitable for storage in our Couchbase bucket: